score-xml/src/main/java/io/cloudslang/content/xml/actions/Remove.java

```diff
@@ -29,28 +34,36 @@
      */
     @Action(name = "Remove",
             outputs = {
-                    @Output(Constants.OutputNames.RESULT_TEXT),
-                    @Output(Constants.OutputNames.RETURN_RESULT),
-                    @Output(Constants.OutputNames.RESULT_XML)},
+                    @Output(Constants.Outputs.RETURN_CODE),
+                    @Output(Constants.Outputs.RETURN_RESULT),
+                    @Output(Constants.Outputs.RESULT_XML)},
             responses = {
-                    @Response(text = Constants.ResponseNames.SUCCESS, field = Constants.OutputNames.RESULT_TEXT, value = Constants.SUCCESS, matchType = MatchType.COMPARE_EQUAL),
-                    @Response(text = Constants.ResponseNames.FAILURE, field = Constants.OutputNames.RESULT_TEXT, value = Constants.FAILURE, matchType = MatchType.COMPARE_EQUAL, isDefault = true, isOnFail = true)})
+                    @Response(text = Constants.ResponseNames.SUCCESS, field = Constants.Outputs.RETURN_CODE, value = Constants.ReturnCodes.SUCCESS, matchType = MatchType.COMPARE_EQUAL),
+                    @Response(text = Constants.ResponseNames.FAILURE, field = Constants.Outputs.RETURN_CODE, value = Constants.ReturnCodes.FAILURE, matchType = MatchType.COMPARE_EQUAL, isDefault = true, isOnFail = true)})
     public Map<String, String> execute(
-            @Param(value = Constants.InputNames.XML_DOCUMENT, required = true) String xmlDocument,
-            @Param(value = Constants.InputNames.XPATH_ELEMENT_QUERY, required = true) String xPathQuery,
-            @Param(Constants.InputNames.ATTRIBUTE_NAME) String attributeName,
-            @Param(Constants.InputNames.SECURE_PROCESSING) String secureProcessing) {
+            @Param(value = Constants.Inputs.XML_DOCUMENT, required = true) String xmlDocument,
+            @Param(value = Constants.Inputs.XML_DOCUMENT_SOURCE) String xmlDocumentSource,
+            @Param(value = Constants.Inputs.XPATH_ELEMENT_QUERY, required = true) String xPathQuery,
+            @Param(value = Constants.Inputs.ATTRIBUTE_NAME) String attributeName,
+            @Param(value = Constants.Inputs.SECURE_PROCESSING) String secureProcessing) {
 
-        CommonInputs inputs = new CommonInputs.CommonInputsBuilder()
-                .withXmlDocument(xmlDocument)
-                .withXpathQuery(xPathQuery)
-                .withSecureProcessing(secureProcessing)
-                .build();
+        Map<String, String> result = new HashMap<>();
+        try {
```

Comment:
why add this try catch if the same thing is already done in the execute method from RemoveService?